akka-bench-jmh/src/main/scala/akka/actor/ForkJoinActorBenchmark.scala

```diff
@@ -72,6 +72,56 @@ class ForkJoinActorBenchmark {
 
   @Benchmark
   @Measurement(timeUnit = TimeUnit.MILLISECONDS)
+  @OperationsPerInvocation(totalMessagesLessThanCores)
+  def pingPongLessActorsThanCores(): Unit = {
+    val pingPongs =
+      for {
+        i <- 1 to lessThanCoresActorPairs
+      } yield {
+        val ping = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        val pong = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        (ping, pong)
+      }
+
+    pingPongs.foreach { case (ping, pong) => ping.tell(message, pong) }
+
+    pingPongs.foreach {
+      case (ping, pong) =>
+        val p = TestProbe()
+        p.watch(ping)
+        p.expectTerminated(ping, timeout)
+        p.watch(pong)
+        p.expectTerminated(pong, timeout)
+    }
+  }
+
+  @Benchmark
+  @Measurement(timeUnit = TimeUnit.MILLISECONDS)
+  @OperationsPerInvocation(totalMessagesMoreThanCores)
+  def pingPongMoreActorsThanCores(): Unit = {
+    val pingPongs =
+      for {
+        i <- 1 to moreThanCoresActorPairs
+      } yield {
+        val ping = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        val pong = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        (ping, pong)
+      }
+
+    pingPongs.foreach { case (ping, pong) => ping.tell(message, pong) }
```

Comment:
should we have more messages in flight?
the default throughput setting for the dispatcher is 5, and that influence the overall throughput a lot, see http://letitcrash.com/post/20397701710/50-million-messages-per-second-on-a-single